common/io.cpp

```diff
@@ -71,15 +71,6 @@ ExpectedSize ByteWriter::Write(
 }
 
 
-ExpectedSize StreamWriter::Write(
```

Comment:
Q:

Shouldn't we extend this instead, to be the `ostream` part of what you have jsut done with `istream`, so that writers also can map to streams?

Reply:
If it turns out we need it then yes. Right now I'm not aware of any place where we would need an `ostream` for a given `Writer`. In fact, we don't use any real (not just shoveling data between buffers) `Writer` at all, AFAICT.

Reply:
I'm using `Writer` quite a bit in the HTTP lib. But not `StreamWriter`, indeed.

Reply:
But also just shoveling data between buffers, right? :wink:

Reply:
We don't know. In the tests it does, but the `Writer` will likely be a pipe to mender-flash in the future.